pkg/profiling/mdoule.go

```diff
@@ -0,0 +1,77 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
The file name is `mdoule`?

Reply:
Yes, It's declared as the profiling module. Management of the profiling module life cycle.